src/components/results.jsx

```diff
@@ -29,7 +29,7 @@ export default React.createClass({
   renderTable() {
     return <Grid className="results-section">
       <Row>
-        <Col>
+        <Col md={8} mdOffset={2} xs={12} >
```

Comment:
xs={12} se pune by default. La fel si sm={12}. Regula asta in schimb o sa mai puna lg={8}
TLDR sterge xs.

LGTM in rest